bgpd/bgp_attr.h

```diff
@@ -555,6 +555,10 @@ static inline void bgp_attr_set_community(struct attr *attr,
 		SET_FLAG(attr->flag, ATTR_FLAG_BIT(BGP_ATTR_COMMUNITIES));
 	else
 		UNSET_FLAG(attr->flag, ATTR_FLAG_BIT(BGP_ATTR_COMMUNITIES));
+
+	if (ecommunity_select_color(attr->ecommunity))
```

Comment:
Why do you want to care about extended communities under regular community functions?

Reply:
mistake from my side. ecommunity!